composer.json

```diff
@@ -27,7 +27,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
Should be `^7.3 || ~8.0.0 || ~8.1.0`.

We have no idea about `8.2.0` or newer (yet).